README.md

```diff
@@ -87,12 +84,12 @@ Yes, now you have a basic stuff working on your desktop but the code is in a ear
   or in the [LICENSE](LICENSE) file, in the root folder of this repository.
 
```

Comment:
```suggestion
```

Reply:
I didn't understand this change, is there a blank space?

Reply:
yes